auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -104,71 +136,217 @@ import Vue from 'vue'
+      business.nameRequest?.enableIncorporation &&
+      supportedEntityFlags.includes(business.nameRequest?.legalType)
+  }
+
+  /** Returns the Name value for the affiliation */
+  private name (item: Business): string {
+    return this.isNumberedIncorporationApplication(item) ? 'Numbered Benefit Company' : item.name
+  }
+
+  /** Returns true if the Name Request is approved */
+  private isApprovedName (name: Names): boolean {
+    return name.state === NrState.APPROVED
+  }
+
+  /** Returns true if the Name Request is rejected */
+  private isRejectedName (name: Names): boolean {
+    return name.state === NrState.REJECTED
+  }
+
+  /** Returns the identifier for the affiliation */
+  private number (item: Business): string {
+    switch (true) {
+      // This case is subject to change once the NR Number is available in the Affiliation Response ALWAYS
+      case (this.isNumberedIncorporationApplication(item)):
+        return 'Pending'
+      case (this.isTemporaryBusinessRegistration(item.corpType.code)):
+        return 'Pending'
+      case this.isNameRequest(item.corpType.code):
+        return item.nameRequest.nrNumber
+      default:
+        return item.businessIdentifier
+    }
+  }
+
+  /** Returns the type of the affiliation */
+  private type (item: Business): AffiliationTypes {
+    switch (true) {
+      case (this.isNameRequest(item.corpType.code)):
+        return AffiliationTypes.NAME_REQUEST
+      case this.isTemporaryBusinessRegistration(item.corpType.code):
+        return AffiliationTypes.INCORPORATION_APPLICATION
+      default:
+        return AffiliationTypes.CORPORATION
+    }
+  }
+
+  /** Returns the corp full description for display */
+  private typeDescription (corpType: CorpTypeCd): string {
+    return GetCorpFullDescription(corpType)
+  }
+
+  /** Returns the status for the affiliation */
+  private status (item: Business): string {
+    switch (true) {
+      case (this.isNameRequest(item.corpType.code)):
+        // Format name request state value for Display
+        return item.nameRequest.state.charAt(0).toUpperCase() + item.nameRequest.state.slice(1).toLowerCase()
+      case this.isTemporaryBusinessRegistration(item.corpType.code):
+        return BusinessState.DRAFT
+      default:
+        return BusinessState.ACTIVE
+    }
+  }
+
+  /** Returns the folio number or a default message */
+  private folio (item: Business): string {
+    return item.folioNumber || this.$t('notAvailable').toString()
+  }
+
+  /** Returns the last modified date string or a default message */
+  private lastModified (item: Business): string {
+    return item.lastModified
+      ? new Date(item.lastModified).toLocaleDateString('en-CA', {
+        timeZone: 'America/Vancouver'
+      })
+      : this.$t('notAvailable').toString()
+  }
+
+  /** Returns the modified by value or a default message */
+  private modifiedBy (item: Business): string {
+    switch (true) {
```

Comment:
may be if else will be better than switch in this case

Reply:
For sure, can do 👍

Reply:
updated in next commit